### PR TITLE
Use latest Liberty Arquillian plugins and update sample projects to Jakarta EE 9

### PR DESCRIPTION
**Pull request by kathrynkodama**
- Update Liberty Arquillian plugins to latest Jakarta EE 9 versions
- Namespace change to: `arquillian-liberty-xx-jakarta-xx`
- Update sample projects `it/managed-tests` and `gradle-tests` to Jakarta EE 9
- Bump version to `2.0.0-SNAPSHOT`
Signed-off-by: Kathryn Kodama <kathryn.s.kodama@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,11 +117,11 @@ Step 3: Add a `dependencyManagement` block that contains the `arquillian-bom`:
...
-        mavenBom "org.jboss.arquillian:arquillian-bom:1.1.13.Final"
+        mavenBom "org.jboss.arquillian:arquillian-bom:1.7.0.Alpha9"
     }
 }
 ```
 
 You can use the command `gradle dependencies` to view transitive dependencies and their versions. 
 
-The previous example uses version `1.1.13.Final` of the `arquillian-bom`. A full Gradle example is available in the `gradle-tests` folder. 
+The previous example uses version `1.7.0.Alpha9` of the `arquillian-bom`. A full Gradle example with Jakarta EE 9 is available in the `gradle-tests` folder.
```
Comment (cherylking) [inline README.md:127]: Should they use a different version for Java EE 8 as implied earlier in the readme?

File: arquillian-liberty-managed-jakarta-junit/pom.xml [hunk 2]
```diff
@@ -3,11 +3,11 @@
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>io.openliberty.arquillian</groupId>
-		<artifactId>arquillian-liberty-dependencies</artifactId>
-		<version>1.1.7-SNAPSHOT</version>
+		<artifactId>arquillian-liberty-jakarta-dependencies</artifactId>
```
Comment (cherylking) [inline arquillian-liberty-managed-jakarta-junit/pom.xml:6]: I see you moved the pom.xml into a differently named folder. This is not consistent with what was done in liberty-arquillian repo.

File: gradle-tests/src/main/java/org/arquillian/example/Greeter.java [hunk 3]
```diff
@@ -18,8 +18,8 @@
 package org.arquillian.example;
 
 import java.io.PrintStream;
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
+import jakarta.enterprise.context.ApplicationScoped;
```
Comment (cherylking) [inline gradle-tests/src/main/java/org/arquillian/example/Greeter.java:21]: Should the copyright year be updated for these test changes?

**Review by cherylking — CHANGES_REQUESTED** [review]

File: arquillian-liberty-managed-jakarta-junit/pom.xml (hunk 2 above)
Comment (kathrynkodama) [reply arquillian-liberty-managed-jakarta-junit/pom.xml]: I've reverted the renaming to stay consistent with the liberty-arquillian repo. Only the artifact ids have changed to use the `-jakarta-` suffix. The folder names have stayed the same.

File: gradle-tests/src/main/java/org/arquillian/example/Greeter.java [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2012, 2018, IBM Corporation, Red Hat Middleware LLC, and individual contributors
+ * Copyright 2012, 2018, 2021, IBM Corporation, Red Hat Middleware LLC, and individual contributors
```
Comment (cherylking) [inline gradle-tests/src/main/java/org/arquillian/example/Greeter.java:2]: When updating copyright years you are supposed to change the last year, not add another year.

**Review by cherylking — CHANGES_REQUESTED** [review]

**Review by cherylking — APPROVED** [review]

**Pull request merged**